db/users.js

```diff
@@ -0,0 +1,115 @@
+const { client } = require("./client");
+const bcrypt = require("bcrypt");
+const SALT_COUNT = 10;
+
+async function createUser({
+    username,
+    password,
+    firstName,
+    lastName,
+    email,
+    imageURL,
+    isAdmin,
+}) {
+    try {
+        const hashedPassword = await bcrypt.hash(password, SALT_COUNT);
+        const {
+            rows: [user],
+        } = await client.query(
+            `
+        INSERT INTO users(username, password, "firstName", "lastName", email, "imageURL", "isAdmin") 
```

Comment:
What happens if someone inserts undefined in any of these...?  Just check and see what happens... might want to error handle?